source/AI.cpp

```diff
@@ -449,6 +449,8 @@ void AI::UpdateEvents(const list<ShipEvent> &events)
 
 		if(event.Actor())
 		{
+			if(event.Type() & ShipEvent::STOPPED_SCANNING)
```

Comment:
~~I know this is my code~~ but could you just add that this removes the SCANNING ship event?

Reply:
I can do that

Reply:
I added a comment.